hera_cal/abscal_funcs.py

```diff
@@ -585,6 +585,90 @@ def delay_slope_lincal(model, data, antpos, wgts=None, df=9.765625e4, medfilt=Tr
 
     return fit
 
+def global_phase_slope_logcal(model, data, antpos, wgts=None, verbose=True, tol=1.0):
+    """
+    Solve for a frequency-independent spatial phase slope using the equation 
+
+    median_over_freq(V_ij,xy^data / V_ij,xy^model) = dot(Phi_x, r_i) - dot(Phi_y, r_j)
```

Comment:
is there supposed to be an `angle` somewhere on the LHS of this eqn?

Reply:
Good call. Fixed.